apps/src/tutorialExplorer/tutorial.jsx

```diff
@@ -50,27 +76,20 @@ const Tutorial = React.createClass({
           closeClicked={this.tutorialDetailClosed}
         />
         <div
-          className="col-33"
-          style={{float: 'left', padding: 2}}
+          style={tutorialOuterStyle}
+          onClick={this.tutorialClicked}
         >
-          <div style={{padding: 5}}>
-            <div
-              style={{cursor: 'pointer'}}
-              onClick={this.tutorialClicked}
-            >
-              <div style={{position: "relative", width: "100%", height: 0, paddingTop: "75%"}}>
-                <img
-                  src={this.props.item.image.replace("/images/", "/images/fill-480x360/").replace(".png", ".jpg")}
-                  style={{position: "absolute", top: 0, left: 0, width: "100%"}}
-                />
-              </div>
-              <div style={styles.tutorialName}>
-                {this.props.item.name}
-              </div>
-              <div style={styles.tutorialSub}>
-                {getTutorialDetailString(this.props.item)}
-              </div>
-            </div>
+          <div style={styles.tutorialImageContainer}>
+            <img
+              src={this.props.item.image.replace("/images/", "/images/fill-480x360/").replace(".png", ".jpg")}
```

Comment:
The gsheet lists .png files but by changing them to .jpg Pegasus will resize them before serving, and we save a bunch of bandwidth.